### PR TITLE
Add the `GetLoadingProgress` RPC

### PR DESCRIPTION
**Pull request by SimFG**
Add the `GetLoadingProgress ` RPC, which is sourced from the `ShowCollections` and `ShowPartitions` RPC when the request param `ShowType` is `ShowType_InMemory`.

issue: #19445
Signed-off-by: SimFG <bang.fu@zilliz.com>

### CODE REVIEW

**Comment by SimFG** [comment]
rerun ut

File: internal/proxy/impl.go [hunk 1]
```diff
@@ -1697,6 +1697,138 @@ func (node *Proxy) ShowPartitions(ctx context.Context, request *milvuspb.ShowPar
...
+	return progress, nil
+}
+
+func (node *Proxy) GetLoadingProgress(ctx context.Context, request *milvuspb.GetLoadingProgressRequest) (*milvuspb.GetLoadingProgressResponse, error) {
+	if !node.checkHealthy() {
+		return &milvuspb.GetLoadingProgressResponse{Status: unhealthyStatus()}, nil
+	}
+	method := "GetLoadingProgress"
+	tr := timerecord.NewTimeRecorder(method)
+	log.Debug(method, zap.String("collection_name", request.CollectionName),
```
Comment (xiaofan-luan) [inline internal/proxy/impl.go:1776]: use at least info log in all DDL path. and the log format should be something like:
	log.Info(
		rpcReceived(method),
		zap.String("traceID", traceID),
		zap.String("role", typeutil.ProxyRole),
		zap.Any("request", request))

File: internal/proxy/impl.go [hunk 2]
```diff
@@ -1697,6 +1697,138 @@ func (node *Proxy) ShowPartitions(ctx context.Context, request *milvuspb.ShowPar
...
+		if progress, err = node.getCollectionProgress(ctx, request, collectionID); err != nil {
+			return getErrResponse(err), nil
+		}
+	} else {
+		if progress, err = node.getPartitionProgress(ctx, request, collectionID); err != nil {
+			return getErrResponse(err), nil
+		}
+	}
+
+	logger.Debug(method+"success", zap.String("collection_name", request.CollectionName),
```
Comment (xiaofan-luan) [inline internal/proxy/impl.go:1819]: log.Info(
		rpcDone(method),
		zap.String("traceID", traceID),
		zap.String("role", typeutil.ProxyRole),
		zap.Int64("MsgID", cit.ID()),
		zap.Uint64("BeginTs", cit.BeginTs()),
		zap.Uint64("EndTs", cit.EndTs()),
		zap.String("db", request.DbName),
		zap.String("collection", request.CollectionName),
		zap.String("field", request.FieldName),
		zap.Any("extra_params", request.ExtraParams))

also print the percentage int the log out.

**Pull request merged**